src/user/views/adsManager/views/advanced/components/campaign/fields/FormatField.tsx

```diff
@@ -31,6 +47,14 @@ export function FormatField({ prices }: PriceProps) {
           prices={prices}
           name="Newsfeed ads"
         />
+        <ProspectButton
+          name="New tab takeovers"
+          salesLink="https://contact.ads.brave.com/"
+        />
+        <ProspectButton
+          name="Search keyword ads"
```

Comment:
Yes, this is something we need to discuss, specifically naming , it's shown as "Search keyword ads" on Brave.com and help center to differentiate between "Search homepage ads", which may not actually launch AFAIK.

Keyword ads in this context referred to the keywords the user types in to find what they're looking for, but I agree it can cause issues since our solution is fully automated and users aren't buying Search. 

Screenshot from ads-help.brave.com, which shows the framework I was working towards was to account for different types of ads on Search. If homepage ads won't be a thing, we can just revert to "search ads" everywhere in marketing too. Open to suggestions/direction here @tackley.
<img width="297" alt="image" src="https://github.com/brave/ads-ui/assets/79997955/45ed5202-2453-4997-b0df-0334080273eb">

Reply:
Going to keep as `Search keyword ads` for now.